src/test-run/commands/actions.js

```diff
@@ -324,13 +324,15 @@ export class NavigateToCommand extends Assignable {
 
         this.type = TYPE.navigateTo;
         this.url  = null;
+        this.storages = null;
 
         this._assignFrom(obj, true);
     }
 
     _getAssignableProperties () {
         return [
-            { name: 'url', type: urlArgument, required: true }
+            { name: 'url', type: urlArgument, required: true },
+            { name: 'storages' }
```

Comment:
check the type here. It should be string. In the same way like in other properties in this and other commands

Reply:
it should be 'type: nonEmptyStringArgument' without the `required` flag.